modules/apps/notification/notification-service/src/main/java/com/liferay/notification/internal/type/EmailNotificationType.java

```diff
@@ -478,6 +482,12 @@ protected void activate(BundleContext bundleContext) {
 				_objectDefinitionLocalService, _objectFieldLocalService,
 				_organizationLocalService, _roleLocalService,
 				_userGroupRoleLocalService, _userLocalService));
+		_init(bundleContext);
```

Comment:
@rijema please inline.. no need to create the `_init` method as it's only used once.

Reply:
Ok